tests/posix/system.icn

```diff
@@ -44,5 +49,10 @@ procedure main()
 #      }
    remove("foo.out")
    remove("x.icn")
+   #
+   # add following remove command to remove the generated file, caused
+   # problem in Makefile
+   #
+   remove("x")
```

Comment:
will this work on Windows, since the generated executable would be  "x.exe" ?

I didn't check to see how much the posix test has coverage on Windows anyway.

Reply:
Something like this would do:
```
$ifdef _MS_WINDOWS_NT
   remove("x.exe")
$else
   remove("x")
$endif
```

Reply:
I have inserted your suggested change here and will push this shortly.